wfcGen.py

```diff
@@ -134,24 +130,24 @@ def loadEntropy(self, grid):
                 if (self.gp[1]+xy)<len(grid):
                     if grid[self.gp[1]+xy][self.gp[0]].collapsed:
                         self.entropy = intersection(self.entropy, grid[self.gp[1]+xy][self.gp[0]].nbg[2 - xy])
-                if (self.gp[1]+xy)<len(grid):
+                if (self.gp[0]+xy)<len(grid):
                     if grid[self.gp[1]][self.gp[0]+xy].collapsed:
-                        self.entropy = intersection(self.entropy, grid[self.gp[1]][self.gp[0]+xy].nbg[2 - xy])
+                        self.entropy = intersection(self.entropy, grid[self.gp[1]][self.gp[0]+xy].nbg[1 - xy])
 
         else:
             self.entropy = [self.id]
 
-    def getEntropy(self):
+    def getEntropy(self, cells):
         if self.collapsed:
             return -1
         else:
             if self.checkCollapsedNeighbours:
-                return len(cells.keys())
+                return 9223372036854775800

```

Comment:
??
Please, at least write a comment to explain
(maybe for the `-1` return value as well)